src/electron/views/automated-checks/automated-checks-view.tsx

```diff
@@ -70,17 +78,26 @@ export class AutomatedChecksView extends React.Component<AutomatedChecksViewProp
                 unifiedScanResultStoreData,
                 cardSelectionViewData.highlightedResultUids,
             );
+            reportHTML = deps.reportGenerator.generateFastPassAutomatedChecksReport(
```

Comment:
I'm not sure this is the best place to generate the report. It looks to me this will generate the report every time we render the `automated-checks-view` component without the user input which may or may not impact performance. Is this the strategy we are using on AI-Web?

Reply:
Agreed. Made the change to consume report export component which was more than I wanted to do but we also get the dialog now 😄